resources/fetchers/monitoring_fetcher.go

```diff
@@ -45,16 +47,39 @@ type MonitoringResource struct {
 	identity *awslib.Identity
 }
 
+type SecurityHubResource struct {
+	securityhub.SecurityHub
+}
+
 func (m MonitoringFetcher) Fetch(ctx context.Context, cMetadata fetching.CycleMetadata) error {
 	m.log.Debug("Starting MonitoringFetcher.Fetch")
 	out, err := m.provider.AggregateResources(ctx)
 	if err != nil {
-		return err
+		m.log.Errorf("failed to aggregate monitoring resources: %v", err)
+	}
+	if out != nil {
+		m.resourceCh <- fetching.ResourceInfo{
+			Resource:      MonitoringResource{*out, m.cloudIdentity},
+			CycleMetadata: cMetadata,
+		}
+	}
+	hubs, err := awslib.MultiRegionFetch(ctx, m.securityhub, func(ctx context.Context, client securityhub.Service) (securityhub.SecurityHub, error) {
```

Comment:
I'm rethinking it, should we generate the provider clients inside `MultiRegionFetch` it might be cleaner...
wdyt?

Reply:
sounds good,lets create another pr with this change